Launcher/Classes/LocalizationHandler.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+
+class LocalizationHandler {
+    
+    let applicationStateHandler = ApplicationStateHandler()
```

Comment:
Maybe this handler should be a shared instance. What does initializing this "mean"?